.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,9 @@
 | Deprecations? | yes/no <!-- please update UPGRADE-*.md and src/**/CHANGELOG.md files -->
 | Tickets       | Fix #... <!-- prefix each issue number with "Fix #", no need to create an issue if none exists, explain below instead -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!-- required for new features -->
+
 <!--
-Replace this notice by a short README for your feature/bugfix.
+Replace this notice by a short description of your feature/bugfix.
```

Comment:
If you want things to be well documented, there's no need to ask for the description to be short.
```suggestion
Replace this notice by a description of your feature/bugfix.
```